README.md

```diff
@@ -21,9 +21,7 @@
   <br/>
 </p>
 
-### Get insights you can act on, fast
-
-<p>
+<p align="center">
```

Comment:
I can't comment on it, but the "Enso Language" header above isn't real text. Could we make it standard GH font?

Reply:
We want to change it to something else (not contain the word "language" as well). This is not part of this PR change, so lets keep it as it for now, but I will create other PRs for improving README in the next days. This should be covered in some of these.